runtime/src/genesis_config_presets.rs

```diff
@@ -250,30 +229,408 @@ fn default_parachains_host_configuration(
+            "5Hb48vxYpQZQHRbzLMak1AKhtuww6wNK1oMFyigysK8zvHyW",
+            "5CpBzkEqzW5RPjDqMMqJuTSkaNrdbo3WxYC8arVjysd96DCN",
+            VALIDATOR_BALANCE,
+            VALIDATOR_BOND,
+        )?,
+        ValidatorData::from_ids(
+            "5G6win2P1ty9X6DYuvfSFkijHGVx8yceVp1uNFntENyu7H4j",
+            "5G6win2P1ty9X6DYuvfSFkijHGVx8yceVp1uNFntENyu7H4j",
+            "5CkhLKX285NbSy4FioEBTwF7Q5X69wmr2d8z58tWr1V8sZK4",
+            VALIDATOR_BALANCE,
+            VALIDATOR_BOND,
+        )?,
+        ValidatorData::from_ids(
+            "5DXDWSgAioftJk5CBhnE1WM7hJVAiCsXALSLxzcV5ouKLo5p",
+            "5DXDWSgAioftJk5CBhnE1WM7hJVAiCsXALSLxzcV5ouKLo5p",
+            "5E6s3Ho4svXABHokXUCcVpZ6UraRR7JpA6VBmYHDau1ZLep1",
+            VALIDATOR_BALANCE,
+            VALIDATOR_BOND,
+        )?,
+        ValidatorData::from_ids(
+            "5C5PKzsRhThtFNbUTeKAaaubFK862Pg75AQM4GavWfghvVK5",
+            "5C5PKzsRhThtFNbUTeKAaaubFK862Pg75AQM4GavWfghvVK5",
+            "5HHKm8QwxoogA1crujuDLQhjxAcGX6VCACqhjYTffBW2nENj",
+            VALIDATOR_BALANCE,
+            VALIDATOR_BOND,
+        )?,
+        ValidatorData::from_ids(
+            "5H6TGvsWATCc6YzJD4iNdeLE54RMvDcy5BgbqamNPJNNbr5L",
+            "5H6TGvsWATCc6YzJD4iNdeLE54RMvDcy5BgbqamNPJNNbr5L",
+            "5G5pgFvQJXiQph63HbtAQey6bNRdzpqbJezt2iAe3xGYtRyt",
+            VALIDATOR_BALANCE,
+            VALIDATOR_BOND,
+        )?,
+        ValidatorData::from_ids(
+            "5CuCBRRajfFFBwRTJ4EPjyPMgoMDKzRSfBa6ktcw8DVcydwM",
+            "5CuCBRRajfFFBwRTJ4EPjyPMgoMDKzRSfBa6ktcw8DVcydwM",
+            "5HdvRwJirHLhC98fkyp22EK5XXfD6dxGe9o45UT49JEES712",
+            VALIDATOR_BALANCE,
+            VALIDATOR_BOND,
+        )?,
+        ValidatorData::from_ids(
+            "5FgWmvTtUMBTKaMa53MgJskPXUwakJwyx61VGxEjP8YPEyQv",
+            "5FgWmvTtUMBTKaMa53MgJskPXUwakJwyx61VGxEjP8YPEyQv",
+            "5GcR9PeGmeFzKpkbVEbP95U4WWSK4z6CWGC9pN6tQpTxbVGS",
+            VALIDATOR_BALANCE,
+            VALIDATOR_BOND,
+        )?,
+        ValidatorData::from_ids(
+            "5C5XGEVmFv7NE6tZ4yJjvCHaaGFxPoTCTURP6WBPxVmppbx2",
+            "5C5XGEVmFv7NE6tZ4yJjvCHaaGFxPoTCTURP6WBPxVmppbx2",
+            "5Cum65qBcju8KnLjxog66kJ7dtbZE92iA3SbDNuL6CPgPqWq",
+            VALIDATOR_BALANCE,
+            VALIDATOR_BOND,
+        )?,
+        ValidatorData::from_ids(
+            "5DeewRqTpr6wm5HZKDRikWpcZ3B6z9ESRFgaqacJEhFAK5Gy",
+            "5DeewRqTpr6wm5HZKDRikWpcZ3B6z9ESRFgaqacJEhFAK5Gy",
+            "5GyiGjs9udrpQEZcS3aFYtHQ7vHHbwkkVEtwce3vHzHpSvZ9",
+            VALIDATOR_BALANCE,
+            VALIDATOR_BOND,
+        )?,
+        ValidatorData::from_ids(
+            "5Cd7HLhDtvzUsmsmBZ7tvGHCdtCofEiAfcVnxWW9U1SyaU9C",
+            "5Cd7HLhDtvzUsmsmBZ7tvGHCdtCofEiAfcVnxWW9U1SyaU9C",
+            "5Euet7r4REQogSfNwBjqvwweFPq7hHsTgaU91dZf2xG9eskf",
+            VALIDATOR_BALANCE,
+            VALIDATOR_BOND,
+        )?,
+    ];
+
+    let nominators = [
```

Comment:
We don't need an initial nominator set for now

Reply:
Done